.github/workflows/mypy_primer_comment.yml

```
@@ -1,11 +1,11 @@
 name: Comment with mypy_primer diff
```

Also, perhaps make it more obvious that it's not going to be triggered anymore:
```suggestion
name: >-
  [Reusable, do not click]
  Comment with mypy_primer diff
```

---

@AlexWaygood after merging this, go to https://github.com/python/mypy/actions and disable this workflow. The inclusion from the other workflow will keep working, but it'll show up lower than the others in the sidebar, which is useful. I'm doing this in many places and got CPython to adopt the same technique which decluttered the sidebar a bit (also with pinning the most important workflows to force them to the top of that list).

---

I'm not a maintainer here, just a triager, so it's out of my power to do this :-) @sobolevn is a mypy maintainer, though

---

Another reason CPython's workflows *had to* be consolidated into a “master workflow” was the ability to integrate alls-green and then use it as a single branch protection check rather than having multiple entry points to manage.